src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1667,6 +1667,41 @@ class Simplex_tree {
     return sh; // None of its faces has the same filtration.
   }
 
+ public:
+  /** \brief This function resets filtration value from a given dimension. Resets all the Simplex_tree when
+   * `min_dim = 0`.
```

Comment:
Maybe we should comment that with `min_dim>0` this may break the filtration property, and it is the user's responsibility to make it a filtration (using a large enough `filt_value`, or calling `make_filtration_non_decreasing` afterwards for instance).

Reply:
Done on 12371aa8

Reply:
"This function resets the filtration value of all the simplices of dimension at least min_dim" might be slightly more explicit.

Reply:
Documentation changed (c++ and python) on e64dd4039e6dff35322ff01a8aa82d79c77e9c8e